package.json

```diff
@@ -145,11 +148,10 @@
   },
   "lint-staged": {
     "*.js": [
-      "eslint --fix",
```

Comment:
Can you revert this to just `eslint --fix` please? Otherwise lint-staged is not going to be able to pass the files to eslint, since the format script is already carrying some filenames.